cellprofiler/gui/cpframe.py

```diff
@@ -972,113 +972,119 @@ def __on_help_path_list(self, event):
+    index.html; (f) Bioformats © 2000-2013 University of Dundee & Open
+    Microscopy Environment; ice © 2013 ZeroC, Inc.; mySQLdb © 2013 Andy
+    Dustman; omero © 2002-2013 Open Microscopy Environment; and ZeroMQ ©
+    2007-2012 iMatix Corporation and Contributors; (g) wxWidgets ©
+    1999-2005 Julian Smart, et al.; and wxPython © 1992-2006 Julian Smart,
+    et al. and © 1996 Artificial Intelligence Applications Institute under
+    the wxWindows license; (h) Intel Visual Fortran Compiler Run-time
+    Library, Intel Fortran Portability Library, Intel OMP Runtime Library,
+    Math Library for Intel Compilers, Short Vector Math Library for Intel
+    Compilers under the Single User license provision for
+    Resdistributables © 2013 Intel Corporation under the End User License
+    Agreement for the Intel Software Development Products; and
+    (i) FastEMD © Copyright (c) 2009-2012, Ofir Pele. under BSD licenses
+    
+    """
+    
+            icon_copyrights = get_icon_copyrights()
+            if additional_copyrights is not None:
+                additional_copyrights += icon_copyrights
+            button_list.append(("Additional licenses and attributions", 
+                                additional_copyrights))
+    
+            for button_text, license_text in button_list:
+                sub_sizer = wx.BoxSizer(wx.VERTICAL)
+                sizer.Add(sub_sizer, 0, wx.EXPAND | wx.ALL, 5)
+                license_button = wx.Button(panel, -1, button_text + " >>")
+                sub_sizer.Add(license_button, 0, wx.ALIGN_LEFT | wx.RIGHT, 3)
+    
+                license_ctl = wx.StaticText(panel, -1, license_text)
+                sub_sizer.Add(license_ctl, 0, 
+                          wx.EXPAND | wx.ALL, 2)
+                license_ctl.Hide()
+    
+                def on_license(event, license_ctl = license_ctl):
+                    if license_ctl.IsShown():
+                        license_ctl.Hide()
+                    else:
+                        license_ctl.Show()
+                    panel.Layout()
+                    panel.SetupScrolling(scrollToTop = False)
+                    panel.ScrollChildIntoView(license_ctl)
+                    panel.Refresh()
+                license_button.Bind(wx.EVT_BUTTON, on_license)
+    
+            sizer.Add(dlg.CreateStdDialogButtonSizer(wx.OK), 0, 
+                      wx.ALIGN_RIGHT | wx.ALL, 5)
+            panel.Layout()
             panel.SetupScrolling()
-            panel.Hide()
-
-            def on_license(event, panel=panel):
-                if panel.IsShown():
-                    panel.Hide()
-                else:
-                    panel.Show()
-                dlg.Fit()
-                dlg.Refresh()
-            license_button.Bind(wx.EVT_BUTTON, on_license)
-
-        sizer.Add(dlg.CreateStdDialogButtonSizer(wx.OK), 0, 
-                  wx.ALIGN_RIGHT | wx.ALL, 5)
-        dlg.Fit()
-        dlg.ShowModal()
+            dlg.Layout()
```

Comment:
Change from dlg.Fit() to dlg.Layout() so dialog conforms to the size specified at the top of this routine.